REACT/tekgain/src/Component/CourseModule/Course.js

```diff
@@ -0,0 +1,139 @@
+import react,{useState} from 'react';
+import Addcourse from './Addcourse';
+import UpdateCourse from './UpdateCourse';
+import ViewCourse from './Viewcourse';
+import ViewFedbacktracking from './ViewFeedbacktracking';
+import Deactivate from './Deactivate';
+
+const Course=(data)=>{
+    
+    const [addCoursebtn,setaddCoursebtn]=useState(false);
+    const [updateCoursebtn,setUpdateCoursebtn]=useState(false);
+    const [viewCoursebtn,setViewCoursebtn]=useState(false);
+    const [viewFeedBackbtn,setViewFeedbackbtn]=useState(false);
+    const [deactivatebtn,setDeactivatebtn]=useState(false)
+    console.log(data)
+    const handleAddcourse=()=>{
+        
+        setaddCoursebtn((currentAddCourse)=>!currentAddCourse)
+        
+        {updateCoursebtn?setUpdateCoursebtn((currentUpdateCourse)=>!currentUpdateCourse):
```

Comment:
remove {} since we are already in JavaScript Mode and use short circuiting instead of ternary operator